modules/ocf_tv/files/X11/xorg.conf

```diff
@@ -1,5 +1,7 @@
 Section "Device"
    Identifier  "Intel Graphics"
    Driver      "intel"
+   Option      "AccelMethod"    "sna"
```

Comment:
Thanks! This is only needed on jessie though. It is the default in jessie-backports and stretch. I'd suggest upgrading.